content/10.diagnostics.md

```diff
@@ -182,61 +211,68 @@ Circulating antibody titers to other coronaviruses have been reported to decline
 At 6 months, 92% of patients were positive for SARS-CoV-2 memory CD4+ T cells.
 Indeed, the spike-specific memory CD4+ T cells abundance over time were similar to the overall SARS-CoV-2-specific CD4+ T cells [@doi:10.1126/science.abf4063].
-T cell immunity to SARS-CoV-2 at 6-8 months post-symptom onset has also been confirmed by others [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636].
-In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never before exposed to SARS-CoV-2. 
-This indicates that there is potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating coronaviruses that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.
-There is concern that immunity may wane over time, as there several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
-Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection
-Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10/fscx] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481]. 
+T cell immunity to SARS-CoV-2 at 6 to 8 months following symptom onset has also been confirmed by other studies [@doi:10.1101/2020.11.01.362319; @doi:10.1101/2020.12.08.416636].
+In another study, T cell reactivity to SARS-CoV-2 epitopes was also detected in some individuals never been exposed to SARS-CoV-2. 
+This finding suggests the potential for cross-reactive T cell recognition between SARS-CoV-2 and pre-existing circulating HCoV that are responsible for the “common cold” [@doi:10.1016/j.cell.2020.05.015], but further research is required.
+
+Concerns have also been raised that immunity may wane over time.
+Several reported cases of reinfection have been confirmed via genomic analysis that revealed distinct variants of SARS-CoV-2 within a single patient [@doi:10/ghfgkt; @doi:10.1093/cid/ciaa1451; @doi:10.1093/cid/ciaa1275].
+Further research is required to determine the full extent to which sustained immunity can be and is typically achieved following SARS-CoV-2 infection.
+Furthermore, it is unclear whether previous infection may carry repercussions in terms of disease severity for patients [@doi:10/fscx] and what implications the possibility of reinfection holds for vaccine development and the long-term efficacy of vaccines [@doi:10.1038/s41591-020-1121-z; @doi:10.3389/fimmu.2020.571481].
```

Comment:
Will generally, take a look at this again soon, few more studies to mention at least, as with everything, studies are being published rapidly on this